subworkflows/nf-neuro/tractoflow/LICENSE

```diff
@@ -0,0 +1,127 @@
+END USER LICENSE AGREEMENT
+
+NOTICE TO USER: PLEASE READ THIS CONTRACT CAREFULLY. BY USING ALL OR ANY PORTION
+OF THE APPLICATION YOU ACCEPT ALL THE TERMS AND CONDITIONS OF THIS AGREEMENT.
+
+1. Definitions
+
+When used in this Agreement, the following terms shall have the respective
+meanings indicated, such meanings to be applicable to both the singular and
+plural forms of the terms defined:
+
+"Licensor" means Sherbrooke Connectivity Imaging Laboratory (SCIL).
+
+"Application" means all of the contents of the files with which this Agreement
+is provided, including but not limited to (i) software; (ii)
+image files ("Databases").
+
+"Use" or "Using" means to access, install, download, copy or otherwise benefit
+from using the functionality of the Application.
+
+“Licensee” means You in an academic or research context.
+
+"Computer" means an electronic device that accepts information in digital or
+similar form and manipulates it for a specific result based on a sequence of
+instructions.
+
+"Modifications" means (a) any file in the Application that results from an addition to,
+deletion from, or modification of the contents; (b) any new file in the Application.
+
+"Contribute" or "Contributor" or "Contributing" means legal entity that contributes to the
+modifications of the Application.
+
+2. Software License
+
+As long as you comply with the terms of this End User License Agreement (the
+"Agreement"), SCIL grants to you a world-wide, royalty-free and non-exclusive
+license to Use the Application. Some third party materials included in the
+Application may be subject to other terms and conditions, which are typically
+listed in "Third party license" part.
+
+2.1 General Use
+
+You may install and Use the Application on your computer. You may contribute to the
+Application. Contributor must respect all terms and conditions of this Agreement.
+
+2.2 Limitations
+
+Using or contributing the Application, (i) you might not use the application for
+commercial purposes ("Commercial Use"); (ii) you must contact the SCIL for a
+patent based on or using the Application.
+
+2.3 Commercial Use
+
+You may not use the application for commercial purposes. For commercial use,
+please contact Imeka Solutions Inc. (https://www.imeka.ca/).
```

Comment:
I am very uncomfortable in applying this part of the license. This is **nf-neuro's implementation of Tractoflow** following the paper and a long lifcycle of updates. It is not the **Tractoflow pipeline** available in its repository, tied for some reasons to Imeka. It is also not an end-user product. I won't agree to strip **nf-neuro** users of using it commercially. And I will dissociate and move things around before binding anything in **nf-neuro** to Imeka.